conf/resources/arrival-response.json

```diff
@@ -168,6 +168,44 @@
+          "messageCorrelationId" : 2
+        },
+        {
+          "dateTime" : "2020-06-03T15:56:00.000Z",
+          "messageType" : "IE044",
+          "message" : "<CC044A>\n<SynIdeMES1>UNOC</SynIdeMES1>\n<SynVerNumMES2>3</SynVerNumMES2>\n<MesSenMES3>SYST17B-NCTS_EU_EXIT</MesSenMES3>\n<MesRecMES6>NCTS</MesRecMES6>\n<DatOfPreMES9>20200603</DatOfPreMES9>\n<TimOfPreMES10>1556</TimOfPreMES10>\n<IntConRefMES11>WE190912102535</IntConRefMES11>\n<AppRefMES14>NCTS</AppRefMES14>\n<TesIndMES18>0</TesIndMES18>\n<MesIdeMES19>1</MesIdeMES19>\n<MesTypMES20>GB044A</MesTypMES20>\n<HEAHEA>\n<DocNumHEA5>19IT02110010007827</DocNumHEA5>\n<IdeOfMeaOfTraAtDHEA78>abcd</IdeOfMeaOfTraAtDHEA78>\n<IdeOfMeaOfTraAtDHEA78LNG>EN</IdeOfMeaOfTraAtDHEA78LNG>\n<NatOfMeaOfTraAtDHEA80>IT</NatOfMeaOfTraAtDHEA80>\n<TotNumOfIteHEA305>1</TotNumOfIteHEA305>\n<TotNumOfPacHEA306>1</TotNumOfPacHEA306>\n<TotGroMasHEA307>1000</TotGroMasHEA307>\n</HEAHEA>\n<TRADESTRD>\n<NamTRD7>The Luggage Carriers</NamTRD7>\n<StrAndNumTRD22>225 Suedopolish Yard,</StrAndNumTRD22>\n<CitTRD24>,</CitTRD24>\n<CouTRD25>GB</CouTRD25>\n<NADLNGRD>EN</NADLNGRD>\n<TINTRD59>GB163910077000</TINTRD59>\n</TRADESTRD>\n<CUSOFFPREOFFRES>\n<RefNumRES1>GB000060</RefNumRES1>\n</CUSOFFPREOFFRES>\n<UNLREMREM>\n<StaOfTheSeaOKREM19>1</StaOfTheSeaOKREM19>\n<UnlRemREM53 xsi:nil=\"true\"/>\n<UnlRemREM53LNG>EN</UnlRemREM53LNG>\n<ConREM65>1</ConREM65>\n<UnlComREM66>1</UnlComREM66>\n<UnlDatREM67>20190913</UnlDatREM67>\n</UNLREMREM>\n<SEAINFSLI>\n<SeaNumSLI2>1</SeaNumSLI2>\n<SEAIDSID>\n<SeaIdeSID1>Seals01</SeaIdeSID1>\n<SeaIdeSID1LNG>EN</SeaIdeSID1LNG>\n</SEAIDSID>\n</SEAINFSLI>\n</CC044A>",
+          "status" : "SubmissionSucceeded",
+          "messageCorrelationId" : 3
+        },
+        {
+          "dateTime" : "2020-06-04T15:56:00.000Z",
+          "messageType" : "IE058",
+          "message" : "<CC058A><SynIdeMES1>UNOC</SynIdeMES1> <SynVerNumMES2>3</SynVerNumMES2> <MesSenMES3>NTA.GB</MesSenMES3> <MesRecMES6>SYST17B-NCTS_EU_EXIT</MesRecMES6> <DatOfPreMES9>20191004</DatOfPreMES9> <TimOfPreMES10>0953</TimOfPreMES10> <IntConRefMES11>22391004095311</IntConRefMES11> <AppRefMES14>NCTS</AppRefMES14> <TesIndMES18>0</TesIndMES18> <MesIdeMES19>22391004095311</MesIdeMES19> <MesTypMES20>GB008A</MesTypMES20> <HEAHEA><DocNumHEA5>19IT02110010007A33</DocNumHEA5> <UnlRemRejDatHEA218>20191004</UnlRemRejDatHEA218></HEAHEA> <FUNERRER1><ErrTypER11>12</ErrTypER11> <ErrPoiER12>SHP.Endorsement date</ErrPoiER12> <OriAttValER14>20190910</OriAttValER14> </FUNERRER1> <FUNERRER1><ErrTypER11>12</ErrTypER11> <ErrPoiER12>INC.Endorsement date</ErrPoiER12> <OriAttValER14>20190910</OriAttValER14> </FUNERRER1> </CC058A>",
```

Comment:
Could we make the rejection mock the correct field (and just have one for now)

`<FUNERRER1>
      <ErrTypER11>12</ErrTypER11>
      <ErrPoiER12>HEA.Identity of means of transport at departure (exp/trans)</ErrPoiER12>
      <OriAttValER14>invalid value</OriAttValER14>
   </FUNERRER1>`